test/integration-test-v2.py

```diff
@@ -118,6 +128,68 @@ def test_wildcard_authz_reuse():
             raise Exception("order for %s included a non-pending authorization (status: %s) from a previous HTTP-01 order" %
                     ((domains), str(authz.body.status)))
 
+def test_revoke_by_issuer():
+    client = make_client(None)
```

Comment:
This stanza is repeated a bunch of times; I think we can reduce it to `auth_and_issue`, right? We might need to make sure `auth_and_issue` returns the order.

Reply:
In the interests of landing this, I'm going to merge as-is, then we can refactor for nicer tests.